packages/tokens/src/themes/bingel/icons.d.ts

```diff
@@ -41,4 +41,19 @@ export declare const icons: {
         description: string;
         svg: string;
     };
+    cirlce: {
```

Comment:
typo

Reply:
<img width="188" alt="image" src="https://user-images.githubusercontent.com/1351956/234295927-8daedf2d-9e74-4213-a992-bb3be76f0046.png">
It's like that in figma :D

Reply:
asked the designers to fix this :D